index.html

```diff
@@ -1,95 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>Accessible Menus</title>
-<link href="https://fonts.googleapis.com/css?family=Raleway:300,400,700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="css/styles.css" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-  </head>
-  <body>
-    <header class="masthead" aria-label="Masthead">
-      <div class="masthead-inner sr-menu-open-hide">
-        <button class="menu-toggle" id="menu-toggle" aria-haspopup="true" aria-label="Open Menu"></button>
-        <a href="https://www.zaneray.com" target="_blank" title="Link to Zaneray.com"><img src="images/zr.png" alt="The ZaneRay Group" class="masthead-logo"></a>
-      </div>
-      <nav id="primary-menu" class="primary-menu" aria-label="Primary Navigation">
-        <ul class="menu-list">
-          <li class="menu-list-item has-submenu">
-            <a href="#shop" class="menu-button" role="button" aria-expanded="false" aria-haspopup="true">Shop</a>
-            <ul class="submenu-list"  aria-expanded="false" aria-hidden="true" aria-label="Shop Subnavigation">
```

Comment:
Do we not need the aria-expanded? I tested and it seems to work w/o it

Reply:
Yes sir. That is what the w3 specification tells us. Lets leave it in for now. Seems to be working.